### PR TITLE
fixed errors found from unittest

### PR DESCRIPTION
**Pull request by rolesucsd**
Fixed a few errors which were identified from the unittest

### CODE REVIEW

File: bakta_queries.py [hunk 1]
```diff
@@ -35,61 +35,58 @@ def get_gene_info_by_name(cursor, gene_name, column):
...
-    Write gene sequences to an output FASTA file.
+    Write gene sequence to FASTA file.
     """
-    with open(output_file_path, "w") as output_file:
-        output_file.write(gene_sequences)
+    with open(output_file_path, 'w') as output_file:
+        for gene_sequence in genes:
+            # Assuming gene_sequence is a string containing the sequence
+            output_file.write(gene_sequence)
+            output_file.write('\n')  # Add a newline to separate sequences
```
Comment (wasade) [inline bakta_queries.py:64]: if `gene_sequence` is a `str` then this is not FASTA format?

Comment (wasade) [reply bakta_queries.py:64]: Can we use the `skbio` writer methods?

Comment (rolesucsd) [reply bakta_queries.py:64]: Yeah, I tried to do this originally, but I can't figure out how to modify the name of each sequence where the header is ">{gene_id}"

Comment (wasade) [reply bakta_queries.py:64]: I recommend `write_gene_sequences_to_fasta` take an iterable like a list of `skbio.DNA` objects. The `.write` method of each object can be used with a provided file. The example below uses a file-like object, but the semantics are consistent with a file

```python
>>> import skbio
>>> example = skbio.DNA('AATTGG', metadata={'id': 'foo'})
>>> import io
>>> x = io.StringIO()
>>> example.write(x)
<_io.StringIO object at 0x7fe58807b4c0>
>>> x.seek(0)
0
>>> x.read()
'>foo\nAATTGG\n'
```

**Pull request merged**